src/content/documentation/develop/onboard-upboard-offboard-users.md

```diff
@@ -126,27 +126,6 @@ Here are some examples of good onboarding pages:
 
 <!-- Two Column Body Module -->
 
-<section class="module">
-<article class="module-content grid-x grid-padding-x align-middle">
-<div class="cell small-12 medium-6 medium-order-2">
-
-![Example onboarding page that details how to use the extension's features and turn them on and off](/assets/img/documentation/develop/Laser_cat_onboarding.png)
```

Comment:
Can we delete the image in the repo too?

Reply:
Good catch, I removed it.